geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-orc/src/main/scala/org/locationtech/geomesa/fs/storage/orc/jobs/OrcSimpleFeatureInputFormat.scala

```diff
@@ -89,7 +89,9 @@ object OrcSimpleFeatureInputFormat {
   }
 
   def getReadColumns(conf: Configuration): Option[Set[Int]] =
-    Option(conf.get(ReadColumnsConfig)).map(_.split(",").map(_.toInt).toSet)
+    Option(conf.get(ReadColumnsConfig))
+      .filter(_.nonEmpty)
```

Comment:
do you know if this is going to make it select all the cols? we should always at least be returning the feature id, right? it might be worthwhile to check out if we are just scanning that col or all cols and discarding them.

Reply:
In the case of SparkSQL, we might return none of the columns (like in a count(*) situation).  Maybe those should be handled better/differently?